test/e2e/storage/persistent_volumes.go

```diff
@@ -313,6 +320,56 @@ var _ = utils.SIGDescribe("PersistentVolumes", func() {
 				pvc = nil
 			})
 		})
+
+		ginkgo.Context("delete pv prior", func() {
+			ginkgo.AfterEach(func() {
+				framework.Logf("AfterEach: Cleaning up test resources.")
+				if errs := e2epv.PVPVCCleanup(c, ns, pv, pvc); len(errs) > 0 {
+					framework.Failf("AfterEach: Failed to delete PVC and/or PV. Errors: %v", utilerrors.NewAggregate(errs))
+				}
+			})
+
+			// Waiting for the pvc and pvc bounded and then delete the pv prior
```

Comment:
bounded -> bound
pvc and pvc -> pvc and pv
delete the pv prior -> delete the pv before pvc